repository/src/main/java/org/alfresco/util/schemacomp/SchemaDifferenceHelper.java

```diff
@@ -142,29 +142,29 @@ protected String describe(Difference difference)
         if (difference.getLeft() == null)
         {
             return I18NUtil.getMessage(
-                        "system.schema_comp.diff.target_only",
-                        ENGLISH,
-                        difference.getRight().getDbObject().getTypeName(),
-                        difference.getRight().getPath(),
-                        difference.getRight().getPropertyValue());
+                    "system.schema_comp.diff.target_only",
+                    ENGLISH,
+                    difference.getRight().getDbObject().getTypeName(),
+                    difference.getRight().getPath(),
+                    difference.getRight().getPropertyValue());
         }
         if (difference.getRight() == null)
         {
             return I18NUtil.getMessage(
-                        "system.schema_comp.diff.ref_only",
-                        ENGLISH,
-                        difference.getLeft().getDbObject().getTypeName(),
-                        difference.getLeft().getPath(),
-                        difference.getLeft().getPropertyValue());
-        }
-
-        return I18NUtil.getMessage(
-                    "system.schema_comp.diff",
+                    "system.schema_comp.diff.ref_only",
                     ENGLISH,
                     difference.getLeft().getDbObject().getTypeName(),
                     difference.getLeft().getPath(),
-                    difference.getLeft().getPropertyValue(),
-                    difference.getRight().getPath(),
-                    difference.getRight().getPropertyValue());
+                    difference.getLeft().getPropertyValue());
+        }
+
+        return I18NUtil.getMessage(
+                "system.schema_comp.diff",
+                ENGLISH,
+                difference.getLeft().getDbObject().getTypeName(),
+                difference.getLeft().getPath(),
+                difference.getLeft().getPropertyValue(),
+                difference.getRight().getPath(),
+                difference.getRight().getPropertyValue());
     }
 }
```

Comment:
No differences here, remove

Reply:
The original formatting is wrong (3 spaces vs 4) but we will keep the original to avoid confusion down the line